src/sentry/api/endpoints/organization_member/details.py

```diff
@@ -372,7 +393,7 @@ def _handle_deletion_by_member(
         acting_member: OrganizationMember,
     ) -> Response:
         # Members can only delete invitations
-        if not member.inviter_id:
+        if not member.is_pending:
```

Comment:
shouldn't this be both checks? inviter id and pending?

Reply:
Updated this because I think `is_pending` is the more correct way of checking if the member is still a pending invite, rather than checking if `inviter_id` still exists